src/utils/getNameForAvatar.ts

```diff
@@ -0,0 +1,6 @@
+import { IAssigneeCombined } from '@/types/interfaces'
+
+export const getNameForAvatar = (user: IAssigneeCombined): string => {
+  const name = user?.familyName || user?.givenName == 'No assignee' ? '' : user?.familyName || user?.givenName
+  return name ?? ''
+}
```

Comment:
We are using multiple `Avatar` components in this app. This particular utils `getNameForAvatar` is misleading IMO. Can we please specify what kind of name does it actually help returning?